Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/03-NodeVariation/01-CreateNodeVariant_ConstraintChecks.feature

```diff
@@ -120,3 +125,11 @@ Feature: Create node variant
       | sourceOrigin    | {"market":"DE", "language":"gsw"} |
       | targetOrigin    | {"market":"DE", "language":"de"}  |
     Then the last command should have thrown an exception of type "NodeAggregateDoesCurrentlyNotCoverDimensionSpacePoint"
+
+  Scenario: Creating variants of tethered root children is allowed
```

Comment:
shouldt this test be part of the actual test file (not the constraints) _with_ proper assertions?

Reply:
I was a bit unsure about it too, but came to the conclusion that this makes sense; This is the test for the constraint checks, explicitly and only testing that in this case they don't block the command.
I would have moved it to another file if I wanted to test the outcome, which I don't :)

Reply:
okay i see then ill add another test though to test the outcome because its important that the projections handle that as well <(